Core/contentblocker.js

```diff
@@ -218,10 +218,13 @@ var duckduckgoContentBlocking = function() {
 
 	// public 
 	function loadSurrogate(url) {
+		var withoutQueryString = url.split("?")[0]        	
```

Comment:
This makes me a bit nervous. What about all the other (including broken from site owners) permutations?